2022/_includes/homepage/slides.html

```diff
@@ -68,6 +68,29 @@
                                         </div>
                                         -->
 
+                                    </div>       
+                                    <div class="counter-box-general-content-date">
+                                        Rebranded to...
+                                    </div>                             
+                                    <div class="counter-box-general-content-date" style="background:rgba(5, 194, 252, 0.7);">
+                                        <div class="counter-box-general-content-date-left">
+                                            <div class="counter-box-general-content-date-left-left"></div>
+                                            <div class="counter-box-general-content-date-left-top"></div>
+                                            <div class="counter-box-general-content-date-left-bottom"></div>
+                                        </div>
+                                        <div class="counter-box-general-content-date-center">
+                                            <div class="counter-box-general-content-date-days" style="text-shadow: 2px -2px white; color:black">DevBcn</div>
+                                            <div class="counter-box-general-content-date-monthyear" style="text-shadow: 2px -2px white;">July 3rd to 5th 2023</div>
+                                            <div class="counter-box-general-content-date-monthyear" style="text-shadow: 2px -2px white;">www.devbcn.com</div>
```

Comment:
```suggestion
                                            <div class="counter-box-general-content-date-monthyear" style="text-shadow: 2px -2px white;"><a href="https://www.devbcn.com" target="_blank" rel="noreferrer">devBcn</a></div>
```

Reply:
good point Anyul.
I think it makes totally sense 👍

Reply:
I assume you both know what a DRAFT PR is.
At the moment there's no point on reviewing it.